Formula/ecl.rb

```diff
@@ -12,13 +12,24 @@ class Ecl < Formula
     sha256 "f86b66f182afdc3b7a8e8e51ca3e230d6ccc308d760faae1b2b345fd1ead0ab3" => :yosemite
   end
 
+  depends_on "pkg-config" => :run
   depends_on "gmp"
+  depends_on "bdw-gc"
+  depends_on "libffi"
+  depends_on "libatomic_ops" => :build
+  depends_on "autoconf" => :run
```

Comment:
ah!! right autoconf dependencies is wrong. I'll fix that.
pkg-config is required to deduct the libffi when we call configure and I suspect that configure require that as well.
 I just copied get the :run from the  contributor documentation cookbook (that at the moment is 404 ).